comment/utils.py

```diff
@@ -1,10 +1,11 @@
 from django.conf import settings
+from django.db.models import Q
 from django.core.mail import send_mail
 from django.contrib.auth.models import User
 
 
 def send_comment_to_admin_email(comment):
-    admin = User.objects.filter(is_superuser=True)[0]
+    admin = User.objects.filter(~Q(email=''), is_superuser=True).first()
```

Comment:
Should it work like this? Admin that receives the emails will always be superuser?